front/app/components/AssignBudgetControl/index.tsx

```diff
@@ -3,7 +3,7 @@ import {
   isNilOrError,
   capitalizeParticipationContextType,
 } from 'utils/helperUtils';
-import { openVerificationModal } from 'components/Verification/verificationModalEvents';
+import { openVerificationModal } from 'containers/App/VerificationModal/verificationModalEvents';
```

Comment:
I think this should live in its own container or maybe in the authentication container? We should think about renaming it in that case but in any case, let's take it out of the app.